community/sync-gateway/1.0.4/scripts/entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+# Set up symlinks to log into a 'normal' location.
+mkdir -p /var/log/sync_gateway
+ln -sf /stdout.log /var/log/sync_gateway/sync_gateway_access.log
+ln -sf /stderr.log /var/log/sync_gateway/sync_gateway_error.log
```

Comment:
@bbrks is the main point of doing the symlink to make the `exec` statement more readable?  Or is `/stdxxx.log` actually needed for anything?

I'm wondering if using a bash variable might also work to make `exec` statement more readable and be slightly simpler.

Reply:
Good suggestion, thanks. It was really just to make it more readable. I'll change to a variable instead.